### ISSUE TITLE
cfillion_Open preferences page - Audio - Device.lua

### ISSUE THREAD

**Issue by bobobo-git** [cfillion/reascripts#2]
don't show the Audo Device Page, but  General Page

**Comment by cfillion** [comment]
Which version of REAPER (and operating system)? It opens the Device page here on v5.99 and v6.34.

**Comment by cfillion** [comment]
This was most likely fixed in version v1.0.3.

```
v1.0.3 by cfillion – April 16 2022
  Fix opening the Device page on Windows and Linux [p=2547777]
```
